bids/layout/layout.py

```diff
@@ -861,7 +862,7 @@ def get(self, return_type='object', target=None, scope='all',
                                      'target entity (\"%s\").' % target)
                 # Construct regex search pattern from target directory template
                 template = self.root + template
-                to_rep = re.findall(r'\{(.*?)\}', template)
+                to_rep = re.findall(r'{(.*?)\}', template)
```

Comment:
This seems like a mistake.

Reply:
Removed this patch set.  Thanks for reviewing.   I'm surprised that the tests all path with the change made.